examples/atari/atari_wrapper.py

```diff
@@ -0,0 +1,228 @@
+import cv2
+import gym
+import numpy as np
+from collections import deque
+
+
+class NoopResetEnv(gym.Wrapper):
+    """Sample initial states by taking random number of no-ops on reset.
+    No-op is assumed to be action 0.
+    :param gym.Env env: the environment to wrap.
+    :param int noop_max: the maximum value of no-ops to run.
+    """
+
+    def __init__(self, env, noop_max=30):
+        super().__init__(env)
+        self.noop_max = noop_max
+        self.noop_action = 0
+        assert env.unwrapped.get_action_meanings()[0] == 'NOOP'
+
+    def reset(self):
+        self.env.reset()
+        noops = self.unwrapped.np_random.randint(self.noop_max) + 1
+        for _ in range(noops):
+            obs, _, done, _ = self.env.step(self.noop_action)
+            if done:
+                obs = self.env.reset()
```

Comment:
Do you need to reset the counter ``_`` if the environment is done?

Reply:
Okay, I find it in [openai baseline](https://github.com/openai/baselines/blob/master/baselines/common/atari_wrappers.py). Do we need to refer to this link in the file?

Reply:
Maybe adding a link on the top of the file is ok.